AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Lists/AdvancedPlaybackListViewController.m

```diff
@@ -91,11 +92,34 @@ - (void)addAllPlayerSelectionOptions {
 - (void)viewDidLoad {
   [super viewDidLoad];
   self.navigationController.navigationBar.translucent = NO;
-  [self.tableView registerNib:[UINib nibWithNibName:@"TableCell" bundle:nil]forCellReuseIdentifier:@"TableCell"];
+    
+  
+    UISwitch *swtLog = [[UISwitch alloc] init];
+    [swtLog addTarget:self action:@selector(changeSwitch:) forControlEvents:UIControlEventValueChanged];
+    UILabel *lblLog = [[UILabel alloc]  initWithFrame:CGRectMake(0,0,44,44)];
+    [lblLog setText:@"QA"];
+    
+    UIBarButtonItem * btn = [[UIBarButtonItem alloc] initWithCustomView:swtLog];
+    UIBarButtonItem * lbl = [[UIBarButtonItem alloc] initWithCustomView:lblLog];
+    self.navigationItem.rightBarButtonItems = [NSArray arrayWithObjects:btn,lbl, nil] ;
+    
+    [self.tableView registerNib:[UINib nibWithNibName:@"TableCell" bundle:nil]
+     forCellReuseIdentifier:@"TableCell"];
 
   [self addAllPlayerSelectionOptions];
 }
 
+     - (void)changeSwitch:(id)sender{
+         if([sender isOn]){
+             NSLog(@"Switch is ON");
+             self.qaLogEnabled=YES;
```

Comment:
use spaces to separate the = sign and the declarations, e.g. `self.qaLogEnabled = YES;`